internal/common/debug/example_test.go

```diff
@@ -75,6 +77,9 @@ func (ati *activityTrackerImpl) Stats() Activities {
 			}{Info: a, Count: count})
 		}
 	}
+	sort.Slice(activities, func(i, j int) bool {
+		return strings.Compare(strings.ToLower(activities[i].Info.ActivityType), strings.ToLower(activities[j].Info.ActivityType)) < 0
```

Comment:
Sorting makes perfect sense, but there's no reason to `ToLower` here afaict.  It'll just make things more ambiguous than they are IRL, where strings need to be identical in both workers and example-test output checks.

Reply:
changed